src-java/swmanager-topology/swmanager-storm-topology/src/main/java/org/openkilda/wfm/topology/switchmanager/service/impl/CommandBuilderImpl.java

```diff
@@ -121,10 +129,60 @@ public CommandBuilderImpl(PersistenceManager persistenceManager, FlowResourcesCo
         return commands;
     }
 
-    private List<BaseInstallFlow> buildInstallDefaultRuleCommands(SwitchId switchId, List<Long> switchRules) {
+    /**
+     * Some default rules require additional properties to be installed. This method filters such rules.
+     */
+    private static boolean isDefaultRuleWithSpecialRequirements(long cookie) {
+        return cookie == SERVER_42_OUTPUT_VLAN_COOKIE
+                || cookie == SERVER_42_OUTPUT_VXLAN_COOKIE;
+    }
+
+    /**
+     * Some default rules require additional properties to be installed. This method creates commands for such rules.
+     */
+    private List<BaseInstallFlow> buildInstallSpecialDefaultRuleCommands(SwitchId switchId, List<Long> switchRules) {
+        SwitchProperties properties = switchPropertiesRepository.findBySwitchId(switchId).orElseThrow(
+                () -> new IllegalStateException(format("Switch properties not found for switch %s", switchId)));
+
         List<BaseInstallFlow> commands = new ArrayList<>();
+        for (Long cookie : switchRules) {
+            InstallServer42FlowBuilder command = InstallServer42Flow.builder()
+                    .transactionId(transactionIdGenerator.generate())
+                    .cookie(cookie)
+                    .switchId(switchId)
+                    .multiTable(properties.isMultiTable())
+                    .inputPort(0)
+                    .outputPort(0)
+                    .server42MacAddress(properties.getServer42MacAddress());
+
+            if (cookie == SERVER_42_OUTPUT_VLAN_COOKIE) {
+                commands.add(command
+                        .id("SWMANAGER_SERVER_42_OUTPUT_VLAN_RULE_INSTALL")
+                        .outputPort(properties.getServer42Port())
+                        .build());
+            } else if (cookie == SERVER_42_OUTPUT_VXLAN_COOKIE) {
+                commands.add(command
+                        .id("SWMANAGER_SERVER_42_OUTPUT_VXLAN_RULE_INSTALL")
+                        .outputPort(properties.getServer42Port())
+                        .build());
```

Comment:
.outputPort(properties.getServer42Port()) could not be moved above because `if() else` section will be extended in next patch PRs